packages/shared-metrics/src/name.js

```diff
@@ -20,24 +20,26 @@ exports.payloadPrefix = 'name';
 // @ts-ignore
 exports.currentPayload = undefined;
 
-const MAX_ATTEMPTS = 60;
-const DELAY = 1000;
+exports.MAX_ATTEMPTS = 10;
```

Comment:
_question:_ Any specific reason for reducing it to 10? I don't have a strong opinion at all, 60 was probably totally arbitrary to begin with and might well be not very reasonable, just asking.

Reply:
lol It should be 60